scene/main/viewport.cpp

```diff
@@ -4902,12 +4928,21 @@ void Viewport::_bind_methods() {
 	BIND_ENUM_CONSTANT(VRS_TEXTURE);
 	BIND_ENUM_CONSTANT(VRS_XR);
 	BIND_ENUM_CONSTANT(VRS_MAX);
+
+	BIND_ENUM_CONSTANT(VRS_UPDATE_DISABLED);
+	BIND_ENUM_CONSTANT(VRS_UPDATE_ONCE);
+	BIND_ENUM_CONSTANT(VRS_UPDATE_ALWAYS);
+	BIND_ENUM_CONSTANT(VRS_UPDATE_MAX);
```

Comment:
Maybe the `*_MAX` members don't have to be exposed. Are they exposed in other cases for some reason?

Reply:
Indeed, I do this mostly out of habit

Reply:
_MAX is exposed so you can do things like this:

```gdscript
# Do something for each VRS update mode constant.
for i in VRS_UPDATE_MAX:
    ...
```

```gdscript
# Cycle VRS update modes.
vrs_update = wrapi(vrs_update + 1, 0, VRS_UPDATE_MAX)
```